20231114_external-mass-tracers/README.md

```diff
@@ -40,7 +40,9 @@ Currently, we generate the S4 lensing map as a sum of the input lensing converge
 This lensing map is provided just for the debugging purpose. 
 LiteBIRD CMB lensing map generated in the same way as the S4 lensing map is also provided for a reference. 
 
-To read these data, please import pickle and use e.g. `pickle.load(open({path to file},"rb"))`.
+To read these data, please import pickle and use e.g. `pickle.load(open({path to file},"rb"))`. 
+Note that this file format is requested by the current delensing pipeline of LiteBIRD, which partially uses Fortran code in addition to Python. 
+The pipeline assumes a file in the pickle format that contains the array of the harmonic coefficients in the Healpix format rather than the Healpy one. 
```

Comment:
if someone wanted to use this with healpy, how would they do that?

Reply:
I have no idea. Is there any function that can transform the array of the harmonic coefficients of the Healpix format [l,m] to the healpy format [1d array]?

Reply:
yes: https://healpy.readthedocs.io/en/latest/generated/healpy.sphtfunc.Alm.getidx.html
I never use the FORTRAN convention, can you be more specific in the README how that array is indexed? Saying "it is in HEALPix convention" is not enough for most users.